test/FunctionalTests/Microsoft.OData.Edm.Tests/Vocabularies/CoreVocabularyTests.cs

```diff
@@ -566,6 +572,7 @@ public void TestCoreVocabularyEnumType(string typeName, string members, bool isF
         [InlineData("SchemaVersion", "Edm.String", "Schema Reference")]
         [InlineData("ODataVersions", "Edm.String", "EntityContainer")]
         [InlineData("Example", "Org.OData.Core.V1.ExampleValue", "EntityType ComplexType TypeDefinition Term Property NavigationProperty Parameter ReturnType")]
+        [InlineData("IsDelta","Org.OData.Core.V1.Tag","ReturnType Parameter")]

```

Comment:
This new line looks unnecessary.

Reply:
This is testing that the IsDelta term is defined in the vocabulary. Of course, that should also be apparent from comparing the expected text, but that's how the test is designed.